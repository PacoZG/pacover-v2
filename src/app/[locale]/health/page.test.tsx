/* eslint-disabled */
import React from 'react'
import { render } from '@testing-library/react'
import { beforeEach, describe, expect, test, vi } from 'vitest'
import Page from '@/app/[locale]/health/page'

describe('Health Page', () => {
  describe('given configured with dependencies', () => {
    beforeEach(() => {
      window.scrollTo = vi.fn()

      vi.mock('next-intl', () => ({
        useTranslations: () => (key: string) => key,
      }))
    })

    describe('Health Page component', () => {
      test('matches the snapshot', () => {
        const { container } = render(<Page />)

        expect(container).toMatchSnapshot()
      })
    })
  })
})
