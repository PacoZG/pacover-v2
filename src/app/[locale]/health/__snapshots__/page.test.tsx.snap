// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Health Page > given configured with dependencies > Health Page component > matches the snapshot 1`] = `
<div>
  <div
    class="page-layout"
  >
    <div
      class="flex justify-center w-full py-8"
    >
      <div
        class="sticky-header rounded-t-xl"
      >
        ok
      </div>
    </div>
  </div>
</div>
`;
